components/AnalyticsEventTracker.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useEffect, useState } from 'react';
+// import { useWeb3 } from '../contexts/Web3Context';
+
+declare global {
+    interface Window {
+        gtag: any;
+        mixpanel: any;
+    }
+}
+
+type Event = {
+    eventType: "click" | "view" | "impression",
+    eventName: string,
+    data?: any
+}
+
+export interface AnalyticsEventTrackerProps {
+    children?: any,
+    events: Event[]
+  }
```

Comment:
minor thing, spacing

Reply:
Yep. Will be mindful of that going forward.